### ISSUE TITLE
htmlwidgets 1.6.3 breaks sortable v0.5.0 bucket and rank lists

### ISSUE THREAD

**Issue by matt-sd-watson** [ramnathv/htmlwidgets#481]
htmlwidgets v1.6.3 breaks the functionality of the sortable rank and bucket lists in sortable 0.5.0 with the following API calls within a R Shiny app:

```
output$BL <- renderUI({
        bucket_list(
          header = NULL,
          orientation = "horizontal",
          group_name = "bucket_list_group",
          add_rank_list(
            text = NULL,
            labels = labels_scratch,
            input_id = "bl_scratch",
            options = c(multiDrag = TRUE),
            class = c("default-sortable", "cytocellbl")
          ),
          add_rank_list(
            labels = labels_top,
            text = NULL,
            input_id = "bl_top",
            options = c(multiDrag = TRUE),
            class = c("default-sortable", "cytocellbl")
          )
        )
      })
      
    }
```

With the update, the following sortable tables render in Shiny but do not have sortable or draggable functionality between lists.

**Comment by gadenbuie** [comment]
Hi @matt-sd-watson, can you try with the latest development version of htmlwidgets from GitHub? I think you may be encountering another version of #477 which should be fixed by #478.

```r
pak::pak("ramntahv/htmlwidgets")
```

**Comment by cpsievert** [comment]
Should be fixed by v1.6.4, sorry about that!
